defect-location-server/backend/src/main/java/cn/com/devopsplus/dop/server/defect/controller/PredictController.java

```diff
@@ -19,41 +16,41 @@ public class PredictController {
 
     @RequestMapping("/get")
     public JSONObject get(){
-        return predictService.get();
+        //用户ID
+        Integer userId=1;
```

Comment:
Is it correct? UserIDs can be fetched via RequestHeader.

Reply:
This would be helpful for you: [UserController.getUserID](https://github.com/nju-softeng/dop-server/blob/28e532b387ceff56972625964b97641ed8c2d227/defect-location-server/backend/src/main/java/cn/com/devopsplus/dop/server/defect/controller/UserController.java#L11-L14)